rendering_resource_manager_service/utils/custom_logging.py

```diff
@@ -41,11 +60,7 @@ def info(level, message):
     :param level: Message level. Only message with a level higher than LOGGING_LEVEL are processed
     :param message: Message to be logged
     """
-    L.info("[%s] [%s] [%s] %s",
-           str(level),
-           datetime.datetime.now().strftime('%Y-%m-%d %H:%M:%S'),
-           current_thread().getName(),
-           message)
+    print_message('[INFO ]', sys.stdout, level, message)
```

Comment:
Well yes, it's to keep an alignment with DEBUG and ERROR. I think it makes the logs easier to read. What do you think?